code/datums/elements/ai_retaliate.dm

```diff
@@ -3,12 +3,18 @@
  * The AI controller is responsible for doing anything with that information.
  */
 /datum/element/ai_retaliate
+	/// What mob type the parent calls
+	var/mob/living/mob_type
+	/// In what range the parent calls the same mobs to retaliate
+	var/call_range
 
-/datum/element/ai_retaliate/Attach(datum/target)
+/datum/element/ai_retaliate/Attach(datum/target, input_mob_type = null, input_call_range = 7)
 	. = ..()
 	if(!ismob(target))
 		return ELEMENT_INCOMPATIBLE
 
+	mob_type = input_mob_type
+	call_range = input_call_range
```

Comment:
https://github.com/tgstation/tgstation/blob/master/.github/guides/STYLE.md#when-passing-vars-through-new-or-initializes-arguments-use-srcvar

tl;dr: use `src.mob_type = mob_type` instead of `mob_type = input_mob_type`
I mean, this has to change anyways but it's good to point out